### PR TITLE
added print indices command

### PR DESCRIPTION
**Pull request by hkulekci**

### CODE REVIEW

File: Resources/config/commands.yml [hunk 1]
```diff
@@ -10,6 +10,13 @@ services:
         tags:
             - { name: console.command }
 
+    apisearch.print_indices_command:
+        class: Apisearch\Command\PrintIndicesCommand
+        arguments:
+        - "@apisearch.repository_bucket"
```
Comment (cv65kr) [inline Resources/config/commands.yml]: Fix indentations, should looks like in other registers.

File: Command/PrintIndicesCommand.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    protected function runCommand(InputInterface $input, OutputInterface $output)
+    {
+        $repository = $input->getArgument('repository');
+        $appId = $input->getOption('app_id');
+        $indices = $this
+            ->repositoryBucket->findRepository($repository)
+            ->getIndices($appId);
+
+        /**
+         * @var Index
```
Comment (hacfi) [inline Command/PrintIndicesCommand.php]: I think this should be `@var Index $index` !? Personally I would also move it down to line 73 or 74.

Comment (hkulekci) [reply Command/PrintIndicesCommand.php]: `composer fix-code` removes `$index` from there, interestingly. And PrintTokensCommand has a comment on the same position. :) I compare from there.

Comment (cv65kr) [reply Command/PrintIndicesCommand.php]: @hkulekci I think, You should use this command locally, and push fixed code to the repository. This command is used to standardize Your code to other parts of app and should be available in an improved form :)

**Comment by hkulekci** [comment]
This feature will change. We should merge after updated https://github.com/apisearch-io/search-server/issues/144 issue

File: Resources/config/commands.yml (hunk 1 above)
Comment (mmoreram) [reply Resources/config/commands.yml]: :+1:

**Comment by hkulekci** [comment]
@cv65kr the pr updated after changing php-client

**Comment by mmoreram** [comment]
@hkulekci merged! Thanks!

**Pull request merged**